lib/traject/met_csv_config.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/met_csv'
+require_relative 'macros/csv'
+extend Macros::MetCsv
+extend Macros::Csv
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+  provide 'allow_empty_fields', true
+end
+
+# MET Museum
+to_field 'id', column('Object ID')
+to_field 'cho_format', column('Object Name')
+to_field 'cho_creator', generate_creator
+to_field 'cho_spatial', column('City')
+to_field 'cho_has_type', column('Classification')
```

Comment:
Yup - Review the mapping (https://docs.google.com/spreadsheets/d/1Sp7uMHizVX7xN7xN9mm-vgEuESQBovXO-qenAo_TV-w/edit#gid=0) lines 12, 47, 48